lib/fs.js

```diff
@@ -438,7 +438,11 @@ function readFileSync(path, options) {
   return buffer;
 }
 
-function close(fd, callback) {
+function defaultCloseCallback(err) {
+  if (err != null) throw err;
+}
+
+function close(fd, callback = defaultCloseCallback) {
   validateInt32(fd, 'fd', 0);
   callback = makeCallback(callback);
```

Comment:
nit: avoid the `makeCallback` call on `defaultCloseCallback` to remove even more boilerplate:
```suggestion
  if (callback !== defaultCloseCallback)
    callback = makeCallback(callback);
```